src/main/java/uk/gov/hmcts/reform/amapi/config/AccessManagementServiceConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.amapi.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.amlib.AccessManagementService;
+
+@Configuration
+@SuppressWarnings("PMD")
```

Comment:
Why the suppress? Is it maybe the same issue that Remi had?

Reply:
Yes it is